scripts/test-runner.ts

```
@@ -0,0 +1,234 @@
+/**
+ * Test script for CLIRunner
+ * Run with: npx tsx scripts/test-runner.ts
+ */
+
+import pty, { type IPty } from 'node-pty';
+
+type RunnerConfig = {
```

ESLint

[error] Use an `interface` instead of a `type`.

(`@typescript-eslint/consistent-type-definitions`)

---

[error] Expected "args" to come before "command".

(perfectionist/sort-object-types)